packages/amplify-cli-core/API.md

```diff
@@ -376,6 +402,9 @@ export enum CFNTemplateFormat {
     YAML = "yaml"
 }
 
+// @public (undocumented)
+const checkIfAuthExists: () => boolean;
```

Comment:
Do we have to keep it if we create `toolkitExtensions` ?

Reply:
These need to be here because they are referenced below in the `toolkitExtensions` namespace. But they are not exported here. They are only exported in the `toolkitExtensions` namespace.